### PR TITLE
feat: add functionality for "Community" dropdown and minor styling changes

### PR DESCRIPTION
**Pull request by ErickSharp**
Before:
image
image
After:
image
image

### CODE REVIEW

**Comment by NathanInnes** [comment]
Please rebase to resolve conflicts.

**Comment by ErickSharp** [comment]
> Please rebase to resolve conflicts.

Just fixed the conflicts.

**Comment by ErickSharp** [comment]
Hold off on pulling, there are still some issues.

**Comment by ErickSharp** [comment]
All conflicts and issues are fixed now, ready for merge

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Comment by Benjozork** [comment]
forgot to submit this

File: src/components/app/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    );
+}
+export const Dropdown: React.FC<{ titleName?: string, className?: string }> = (props) => {
+    const [shown, setShown] = useState(false);
+    return (
+        <div className={`relative inline-block text-left ${props.className}`}>
+            <div>
+                <button draggable="false" type="button" onClick={() => setShown(old => !old)} className="inline-flex justify-center w-full px-4 py-2 text-xl text-white hover:text-blue-light transition-colors duration-200 focus:outline-none" id="options-menu" aria-haspopup="true" aria-expanded="true">
+                    {props.titleName}
+                    {/*Heroicon name: solid/chevron-down*/}
```
[the remark this answers is not in the document]
Comment (ErickSharp) [reply src/components/app/Dropdown.tsx]: removed comment

File: src/components/app/NavBar.tsx [hunk 2]
```diff
@@ -14,23 +15,32 @@ export function NavBar(): JSX.Element {
...
                     <img onClick={scrollTop} className="subpixel-antialiased h-11" src={logo} alt=""/>
                     <div className="absolute inset-y-0 right-3 flex items-center md:hidden">
-                        <Hamburger handleClick={() => setOpen(prevState => !prevState)}/>
+                        <Hamburger handleClick={() => setOpen(prevState => !prevState)} />
+                    </div>
+                    <div className="flex flex-row items-center">
+                        <NavLinks className="hidden md:block" />
+                        <Dropdown className="-ml-1.5" titleName="Community">
+                            <DropdownItem name="Social Media" link="/socials" />
+                            <DropdownItem name="Development Blog" link="/blog" />
```
[the remark this answers is not in the document]
Comment (ErickSharp) [reply src/components/app/NavBar.tsx]: Was discussed with falcon and sabes, is easily able to be changed in the future.

File: src/components/utils/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    );
+}
+export const Dropdown: React.FC<{ titleName?: string, className?: string }> = (props) => {
+    const [shown, setShown] = useState(false);
+    return (
+        <div className={`relative inline-block text-left ${props.className}`}>
+            <div>
+                <button draggable="false" type="button" onClick={() => setShown(old => !old)} className="inline-flex justify-center w-full px-4 py-2 text-xl text-white hover:text-blue-light transition-colors duration-200 focus:outline-none" id="options-menu" aria-haspopup="true" aria-expanded="true">
+                    {props.titleName}
+                    {/*Heroicon name: solid/chevron-down*/}
```
Comment (NathanInnes) [inline src/components/utils/Dropdown.tsx:16]: One comment left to remove

Comment (ErickSharp) [reply src/components/utils/Dropdown.tsx]: sorry about that, fixed

**Comment by NathanInnes** [comment]
On mobile is also broken, if you can fix it that would be great, hmu if you want help with it 
image

**Comment by NathanInnes** [comment]
Fixed issue for mobile and desktop.

Desktop:
image

Mobile: 
image

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**